### DISCUSSION TITLE
[QUESTION] - Blog page template

### DISCUSSION THREAD

**Discussion by marijang** [infinum/eightshift-frontend-libs#791] [General]
How to create blog page template?

Still not using FSE.

**Comment by iruzevic** [comment]
You can create blog components, make blog template or use single.php add those components and pass props like get_the_title, etc. and you are done. Just like creating a complex block from components

**Comment by marijang** [comment]
yes but then is not user editable...

also i enable block template parts and  then i create template part (in folder parts) and then when i edit template throug site-editor but i me getting error:

theme-block-editor-style-css was added to the iframe incorrectly. Please use block.json or enqueue_block_assets to add styles to the iframe.

It seem like problem in iframe passing the css style
